webpack/__mocks__/foremanReact/common/I18n.js

```diff
@@ -1 +1,3 @@
+export { sprintf } from 'jed';
```

Comment:
Why do we need the real version in a mock?

Reply:
Correct formatting of translated messages: 

```js
sprintf(__('Invalid value, expected one of: %s'), lookupKey.validatorRule)
```

Reply:
Can't we just ignore additional arguments to the sprintf?
We are not really testing the strings, are we? We can just have your example to result in `'Invalid value, expected one of: %s'`, right? we'd just get this in the snapshots, but that should be fine.... Or are we depending on it with some tests?